src/traverse.rs

```diff
@@ -1149,9 +1149,11 @@ fn diff_trait_impls<'tcx>(
     let structural_peq_def_id = tcx.require_lang_item(LangItem::StructuralPeq, None);
     let structural_trait_def_ids = [structural_peq_def_id, structural_teq_def_id];
 
-    for (old_impl_def_id, _) in tcx
+    // FIXME(dario)
+    let _ = id_mapping.get_old_crate();
+    for old_impl_def_id in &[] /* tcx
         .all_trait_implementations(id_mapping.get_old_crate())
```

Comment:
this still needs a fix, i just made it compile by commenting it out.. basically i'm not sure how to reproduce the old behavior, since the `all_trait_implementations` method moved into `librustdoc`.